system/jlib/jstats.h

```diff
@@ -583,9 +595,7 @@ class jlib_decl CRuntimeStatisticCollection
     void rollupStatistics(IContextLogger * target) { rollupStatistics(1, &target); }
     void rollupStatistics(unsigned num, IContextLogger * const * targets) const;
 
-
-    virtual void recordStatistics(IStatisticGatherer & target) const;
-    void getNodeProgressInfo(IPropertyTree &node) const;
+    virtual void recordStatistics(IStatisticGatherer & target, bool clear = false) const;
```

Comment:
I am not sure if the default value should be removed.  It would make it less likely to forget to pass it on.

Reply:
Fixed